product_docs/docs/pgd/5/quickstart/connecting_applications.mdx

```diff
@@ -168,44 +168,40 @@ Type "help" for help.
 bdrdb=# 
 ```
 
-## Creating a Connection URL
+## Creating a connection URL
 
-Many applications use a [Connection URL](https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6) to connect to the database. To create a Connection URL, you need to assemble a string in the format:
+Many applications use a [connection URL](https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6) to connect to the database. To create a connection URL, you need to assemble a string in the format:
 
 ```
 postgresql://<user>@<ipaddressofnode1>:6432,<ipaddressofnode2>:6432,<ipaddressofnode3>:6432/bdrdb
 ```
 
-This format of string can be used with the `psql` command, so if our database nodes were on Ip addresses 192.168.9.10, 192.168.10.10 and 192.168.10.11, we could use:
+This format of the string can be used with the `psql` command, so if your database nodes are on IP addresses 192.168.9.10, 192.168.10.10, and 192.168.10.11, you can use:
```

Comment:
If we don't put ticks for utilities, I guess we should delete them here? 

```suggestion
This format of the string can be used with the psql command, so if your database nodes are on IP addresses 192.168.9.10, 192.168.10.10, and 192.168.10.11, you can use:
```

Reply:
No, because that's a command name.